Sources/MapboxCoreNavigation/TileStore.swift

```diff
@@ -32,7 +32,7 @@ extension TileStore {
                             lock.lock()
 
                             if expected.isValue(),
-                               let value = expected.value as? NSNumber {
+                               let value = expected.value {
```

Comment:
To fix the warning of `conditional downcast from 'NSNumber?' to 'NSNumber' does nothing` during tests.